packages/odf/components/storage-consumers/onboarding-modal.tsx

```diff
@@ -32,50 +91,52 @@ export const ClientOnBoardingModal: ClientOnBoardingModalProps = ({
   closeModal,
 }) => {
   const { t } = useCustomTranslation();
-  const MODAL_TITLE = t('Client onboarding token');
-  const [ticket, setTicket] = React.useState('');
-  const [loading, setLoading] = React.useState(false);
-  const [error, setError] = React.useState(null);
-
-  const onCopyToClipboard = () => {
-    navigator.clipboard.writeText(ticket);
-  };
+  const [token, setToken] = React.useState('');
+  const [inProgress, setInProgress] = React.useState(false);
+  const [error, setError] = React.useState<string>(null);
+  const [quota, setQuota] = React.useState(unlimitedQuota);
```

Comment:
this might not be needed right now, but good to have it...
```suggestion
  const [quota, setQuota] = React.useState({ ...unlimitedQuota });
```

Reply:
constant ref like these could bite us back (so better to prevent such things on early stages itself)

Reply:
just my opinion... plz let me know if u think otherwise...

Reply:
I did it for all `setQuota` calls, so we can do it here, too.

Reply:
Done.